vignettes/VectorControl.Rmd

```diff
@@ -15,117 +15,124 @@ knitr::opts_chunk$set(
-characteristics of the bed nets on each distribution...
-
-```{r}
-bednetparams <- simparams
-
-bednet_events = data.frame(
-  timestep = c(1, 2) * year,
-  name=c("Bednets 1", "Bednets 2")
-)
+Having established a base set of parameters, we can now create a copy of this parameter list and update it to specify a net distribution campaign. In the example below, we distribute bednets once a year for two years, the first round to 80% of the population and the second round to 50% of the population. It is possible to change the characteristics of the bednets for each distribution timestep by modifying the values for `dn0`, `rn`, `rnm`, and `gamman`. Because there are 3 mosquito species specified in `set_species()` above and two timesteps when nets are distributed, the matrices for `dn0`, `rn`, and `rnm` have 3 columns and 2 rows.
+  
+```{r, fig.align='center', fig.height=4, fig.width=6}
+bednetstimesteps <- c(1, 2) * year # The bednets will be distributed at the end of the first and the second year. 
 
 bednetparams <- set_bednets(
-  bednetparams,
-  timesteps = bednet_events$timestep,
-  coverages = c(.8, .8),
-  retention = 5 * year,
-  dn0 = matrix(c(.533, .45), nrow=2, ncol=1),
-  rn = matrix(c(.56, .5), nrow=2, ncol=1),
-  rnm = matrix(c(.24, .24), nrow=2, ncol=1),
-  gamman = rep(2.64 * 365, 2)
+  simparams,
+  timesteps = bednetstimesteps,
+  coverages = c(.8, .5),  # The first round is distributed to 80% of the population and the second round to 50%.
+  retention = 5 * year, # Nets are kept on average 5 years
+  dn0 = matrix(c(.533, .45), nrow=2, ncol=3), # Matrix of death probabilities for each mosquito species over time
+  rn = matrix(c(.56, .5), nrow=2, ncol=3), # Matrix of repelling probabilities for each mosquito species over time
+  rnm = matrix(c(.24, .24), nrow=2, ncol=3), # Matrix of minimum repelling probabilities for each mosquito species over time
+  gamman = rep(2.64 * 365, 2) # Vector of bednet half-lives for each distribution timestep
 )
 
 output <- run_simulation(sim_length, bednetparams)
-
-add_intervention_lines(plot_prevalence(output), bednet_events)
 ```
 
-## Indoor spraying
-
-We can do the same for IRS...
-
-```{r}
-sprayingparams <- simparams
-
-peak <- peak_season_offset(sprayingparams)
-spraying_events = data.frame(
-  timestep = c(1, 2) * year + peak - 3 * month,
-  name=c("Spraying 1", "Spraying 2")
-)
-
-sprayingparams <- set_spraying(
-  sprayingparams,
-  timesteps = spraying_events$timestep,
-  coverages = rep(.8, 2),
-  ls_theta = matrix(c(2.025, 2.025), nrow=2, ncol=1),
-  ls_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1),
-  ks_theta = matrix(c(-2.222, -2.222), nrow=2, ncol=1),
-  ks_gamma = matrix(c(0.008, 0.008), nrow=2, ncol=1),
-  ms_theta = matrix(c(-1.232, -1.232), nrow=2, ncol=1),
-  ms_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1)
-)
+### Plot prevalence
+```{r, fig.align='center', fig.height=5, fig.width=7}
+plot_prev()
+```
+  
+## Visualisation 
```

Comment:
I thinkt his section is very helpful - can we call it something other than visualisation:

Something like:
`Comparing input coverage and population bed net usage`